js/main.js

```diff
@@ -22,6 +22,10 @@ function getLunchInfo(lunchMenu) { // Bypassing CORS using JSONP
   }
 }
 
+function getTimeOfDay(date){
+  return date.getHours()*(60*60*1000) + date.getMinutes()*(60* 1000) + date.getSeconds()*(1000) + date.getMilliseconds();
```

Comment:
What units does this return in? Seconds? Ms? Make the function name more descriptive